### ISSUE TITLE
Multi-branch / unit support?

### ISSUE THREAD

**Issue by repodevs** [hexya-erp/hexya#10]
Is hexya will support multi-branch / unit like OCA operating-unit?

**Comment by npiganeau** [comment]
Hi, thanks for your interest in Hexya. Hexya's framework is intended to work the same way as Odoo's with modules that add functionalities. We also use an very similar API (although the language is different) with recorders, environments, etc. 

So eventually supporting this or that feature will only be a matter of coding a module or porting it from Odoo (or OCA).

**Comment by jackgray72** [comment]
Absolutely! Multi-branch and unit support can greatly enhance flexibility and efficiency in development workflows [squishmallowshalloween](https://halloweensquishmallows.com/). It's definitely worth considering for streamlining your processes.
